messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/jms/bridge/JMSBridgeDefinition.java

```diff
@@ -69,8 +70,8 @@
     private static final String SOURCE_CREDENTIAL_REFERENCE_NAME = "source-" + CredentialReference.CREDENTIAL_REFERENCE;
     private static final String TARGET_CREDENTIAL_REFERENCE_NAME = "target-" + CredentialReference.CREDENTIAL_REFERENCE;
 
-    public static final SimpleAttributeDefinition MODULE = create("module", STRING)
-            .setRequired(false)
+    public static final SimpleAttributeDefinition MODULE = create("module", STRING, true)
+            .setDefaultValue(new ModelNode("org.apache.activemq.artemis:main"))
```

Comment:
`module` must not have a default value.

The module that it references MUST depend on the 2 JMS providers (source and targets).
Using `org.apache.activemq.artemis` is a convenient default value for our integration (where we test the JMS bridge from an embedded Artemis broker to the same one) but it's not correct from a feature point of view.

Reply:
ok, i understand your point of view